src/test/java/org/folio/rest/impl/PatronResourceImplTest.java

```diff
@@ -721,6 +789,66 @@ public final void testPostPatronAccountByIdItemByItemIdHold() {
     logger.info("Test done");
   }
 
+  @Test
+  public final void testPostPatronAccountByIdItemByItemIdPage() {
```

Comment:
I see what you mean about checking the body. We might be able to use a header to indicate which request type we want to return. Hmm...

Reply:
I already had code to examine the body, which seems like the right thing to do rather than adding stuffs to the header.  This code is in the latest commit.